scalafix-tests/input/src/main/scala/test/DisableSyntaxMoreRules.scala

```diff
@@ -29,6 +29,10 @@ object DisableSyntaxMoreRules {
     def foobar(a: Int, b: Int = 1) = ??? // assert: DisableSyntax.defaultArgs
   }
 
+  object FooO {
+    def bar(x: Int = 42) = ??? // assert: DisableSyntax.defaultArgs
```

Comment:
Can we test the caret position and message? See https://github.com/scalacenter/scalafix/blob/50e0771a8939be5a9e8ee8a14cd64c4cbf04c2b7/scalafix-tests/input/src/main/scala/test/Disable.scala#L51 for an example

Reply:
https://github.com/scalacenter/scalafix/pull/569